tweb/src/main/frontend/src/js/vue/components/FaqItem.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <div class="card-body faq faq--vue-collapse" v-if="id">
+        <h4 class="faq__title d-flex align-content-center justify-content-between" v-if="title"
+            @click="showCollapse = !showCollapse"
+            :class="showCollapse ? 'collapsed' : null"
+            :aria-controls="id"
+            :aria-expanded="showCollapse ? 'true' : 'false'">
+            <span>{{title}}</span>
+            <i class="fa fa-chevron-left" :class="showCollapse ? 'collapsed' : null"></i>
+        </h4>
+        <b-collapse :id="id" class="faq__answer" v-model="showCollapse">
+            {{description}}
+            <slot></slot>
+        </b-collapse>
+    </div>
+</template>
+
+<script lang="ts">
+    import {Component, Prop, Vue} from "vue-property-decorator";
+    import {Locations} from "../../classes/Locations";
+    import Utils from "../../classes/Utils";
+    import bCollapse from "bootstrap-vue/es/components/collapse/collapse";
+
+    @Component({
+        components: {
+            bCollapse
+        }
+    })
+    export default class FaqItem extends Vue {
+        @Prop() intercom: string;
+        @Prop() title: string;
+        @Prop() description?: string;
+
+        public showCollapse = false;
+        public id: string = "";
+
+        mounted() {
+            this.id = Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15);
```

Comment:
what is this? why do you need it?

Reply:
The b-collapse needs a unique id, without it doesn't work. This is a quick and easy way to generate a random string.
I'll add it to Utils